app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateEmojiIdFragment.kt

```diff
@@ -131,12 +141,96 @@ class CreateEmojiIdFragment : BaseFragment() {
         })
     }
 
-    @OnClick(R.id.create_emoji_id_continue_btn)
-    fun openAuthActivity() {
-        val intent = Intent(activity, AuthActivity::class.java)
-        intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK
-        startActivity(intent)
-        activity?.finish()
+    @OnClick(R.id.create_emoji_id_btn)
+    fun onCreateEmojiIdButtonClick() {
+        showEmojiWheelAnimation()
+    }
+
+    private fun showEmojiWheelAnimation() {
+        emojiWheelAnimView.playAnimation()
+
+        val fadeOutAnimnAnim = ValueAnimator.ofFloat(1f, 0f)
+        fadeOutAnimnAnim.addUpdateListener { valueAnimator: ValueAnimator ->
+            val alpha = valueAnimator.animatedValue as Float
+            nerdFaceEmoji.alpha = alpha
+            awesomeText.alpha = alpha
+            createYourEmojiIdText.alpha = alpha
+            walletAddressDescText.alpha = alpha
+            createEmojiIdButton.alpha = alpha
+        }
+        fadeOutAnimnAnim.addListener(object : AnimatorListenerAdapter() {
+            override fun onAnimationEnd(animation: Animator?) {
+                super.onAnimationEnd(animation)
+                createEmojiIdButton.visibility = View.GONE
+                awesomeTextBackView.visibility = View.GONE
+                justSecBackView.visibility = View.GONE
+                createEmojiIdTextBackView.visibility = View.GONE
+            }
+        })
+
+        fadeOutAnimnAnim.startDelay = CreateEmojiId.walletCreationFadeOutAnimDelayMs
+        fadeOutAnimnAnim.duration = CreateEmojiId.walletCreationFadeOutAnimDurationMs
+        fadeOutAnimnAnim.start()
+
+        uiHandler.postDelayed(
+            { startYourEmojiIDViewAnimation() },
+            emojiWheelAnimView.duration - CreateEmojiId.awesomeTextAnimDurationMs
+        )
+    }
+
+    private fun startYourEmojiIDViewAnimation() {
+        walletAddressDescText.text =
```

Comment:
"Desc" is a good abbreviation, somewhere else in your code it was "des", can you please find that and make that "desc" too?